locales/en/settings.ftl

```diff
@@ -0,0 +1,28 @@
+# This string is shown beneath each of the user’s email addresses to indicate
+# how many known breaches that email address was found in.
+appears-in-x-breaches =
+  { $breachCount ->
+        [one] Appears in { $breachCount } known breach.
+       *[other] Appears in { $breachCount } known breaches.
+  }
+
+resend-verification = Resend verification email
```

Comment:
I understand these come from existing strings, but we didn't do a good job here, and we can use this work to improve things.

IDs should provide context for localizers, if there's no comment doing it. For example: is this a button, a link, something else? Putting more information, and using a more specific ID based on the context would help localizers, and discourage reuse in different contexts.

In this case I would go with `settings-resend-verification-link`.

This applies to all the strings in this file, in particular to very undescriptive ones like `menu`.
`email-verification-required`: is it a title, a message? These might require different style in different locales.